resources/js/store/user/api/userApi.ts

```diff
@@ -0,0 +1,25 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+const fetchBaseUrl = process.env.MIX_FETCH_BASE_URL;
```

Comment:
For the naming, is it only for fetching? or you will also use this for other HTTP methods?
```suggestion
const fetchBaseUrl = process.env.MIX_BASE_URL;
```

Reply:
@roseaugusto You actually have a point, thank you will change it now.